docs/contributing/development-environment.mdx

```diff
@@ -42,7 +42,7 @@ Next, you'll need to configure your development environment. This should be as s
    ```
 </Step>
 <Step title="Open Mage">
-    Open your browser and go to [http://localhost:3000](http://localhost:3000).
+    When developing frontend, use port 3000. If you only need to develop backend, you can use port 6789. Find these ports at [https://localhost:3000](https://localhost:3000) & [https://localhost:6789](https://localhost:6789).
```

Comment:
for localhost, i believe it should be `http` not `https`

Reply:
thx